src/ShopPaymentTransactionBundle/exceptions/TPkgShopPaymentTransactionException_PaymentHandlerDoesNotSupportTransaction.class.php

```diff
@@ -9,6 +9,6 @@
  * file that was distributed with this source code.
  */
 
-class TPkgShopPaymentTransactionException_PaymentHandlerDoesNotSupportTransaction extends TPkgShopPaymentTransactionException
+class TPkgShopPaymentTransactionException_PaymentHandlerDoesNotSupportTransaction extends TPkgCmsException
```

Comment:
Does TPkgCmsException have advantages here (instead of for example nomal Exception)?

Reply:
i dont know but it was in the chain of the replaced TPkgShopPaymentTransactionException.

Reply:
As far as I can see TPkgCmsException only has support for serializable "context" data (for logging) - which is not used here.

-> You can use Exception directly.

Reply:
ok

Reply:
Sorry. I looked in the code and it needs to be a TPkgCmsException. Otherwise \TPkgShopOrderStatusManagerEndPoint::orderStatusAddedHook() will not catch it.

Reply:
redo